{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -257,6 +257,7 @@
 )
 
 COMPRESS_ENABLED = True
+COMPRESS_OFFLINE = env.bool('COMPRESS_OFFLINE', default=False)
```

Comment:
We need compress_offline for auto-prefixer and other things to work by default. We can't expect frontend dev to put it in .env.